tests/quantization/bnb/test_4bit.py

```diff
@@ -648,3 +648,19 @@ class GPTSerializationTest(BaseSerializationTest):
     """
 
     model_name = "gpt2-xl"
+
+
+@require_bitsandbytes
+@require_accelerate
+@require_torch
```

Comment:
```suggestion
```

Reply:
Thanks! I had it for consistency with other tests - removing now!